src/design-tokens/css/base/media.css

```diff
@@ -10,3 +10,10 @@ img {
   max-width: 100%;
   height: auto;
 }
+
+/**
+ * Align SVG elements to be vertically centered by default
+ */
+svg {
+  vertical-align: middle;
```

Comment:
traject already has this through [Tailwind Preflight](https://tailwindcss.com/docs/preflight#images-are-block-level), and I figured this would be lower-specificity to override if needed. Could set the styling in `Icon.module.css` though